gm4_ziprails/data/ziprails/functions/link.mcfunction

```diff
@@ -1,7 +1,7 @@
 #This function docks minecarts under hooks to assign them a zipline movement direction.
-#Handels initial docking.
+#Handles initial docking.
 
-#@s = @e[tag=gm4_minecart] at @s if block ~ ~1 ~ tripwire_hook[attached=true]
+#@s = @e[type=#ziprails:minecart] at @s if block ~ ~1 ~ tripwire_hook[attached=true]

```

Comment:
Should this be `#gm4:minecarts` instead of `#ziprails:minecarts`. That way the same tag can be used for vertical rails.